File: geometry/arc_angle/java/ArcAngle.java
public class ArcAngle {

    /**
     * Will return arc angle
     * @param arc lenght
     * @param radius of circle
     * @return angle
     */
    public float getArcAngle(float arcLenght, float radius) {
        return (360*arcLenght)/(Math.PI*2*radius);
    }

}

File: geometry/arc_angle/javascript/arc_angle.js
/*
    arc_angle
    
    @input: radius of circle, length of arc
    @output: angle of arc
*/

function arc_angle(arc_len, radius){
    let angle = (360*parseFloat(arc_len))/(Math.PI*2*parseFloat(radius))
    
    return angle
}

console.log(arc_angle(1,2));